website/content/docs/security/acl/acl-policies.mdx

```diff
@@ -0,0 +1,435 @@
+
+```shell-session
+ID:           <auto-generated ID>
+Name:         my-app-policy
+Description:  Human-readable description of my policy
+Datacenters:  <specified with the -valid-datacenter option>
+Rules:
+<rules defined in the rules.hcl file>
+```
+
+You can can define several attributes that attach additional metadata and specify the scope of the policy. See [Policy Attributes](#policy-attributes) for details.
+
+### HTTP API Endpoint
+
+The endpoint takes data formatted in HCL or JSON. Refer to the [ACL HTTP API endpoint documentation](/api/acl/acl) for details about the API.
+
+The following example adds a set of rules to a policy called `my-app-policy`. The policy defines access to the `key` resource (Consul K/V). The rules are formatted in HCL, but they are wrapped in JSON so that the data can be sent using cURL:
+
+```shell-session
+$ curl \
+    --request PUT \
+    --data \
+'{
+  "Name": "my-app-policy",
+  "Rules": "key \"\" { policy = \"read\" } key \"foo/\" { policy = \"write\" } key \"foo/private/\" { policy = \"deny\" } operator = \"read\""
+}' http://127.0.0.1:8500/v1/acl/policy?token=<token with ACL "write" access>
+```
+
+The following call performs the same operation as the previous example using JSON:
+
+```shell-session
+$ curl \
+    --request PUT \
+    --data \
+'{
+  "Name": "my-app-policy",
+  "Rules": "{\"key\":{\"\":{\"policy\":\"read\"},\"foo/\":{\"policy\":\"write\"},\"foo/private\":{\"policy\":\"deny\"}},\"operator\":\"read\"}"
+}' http://127.0.0.1:8500/v1/acl/policy?token=<management token>
+```
+
+The policy configuration is returned when the call is succesfully performed:
+
+```json
+{
+  "CreateIndex": 7,
+  "Hash": "UMG6QEbV40Gs7Cgi6l/ZjYWUwRS0pIxxusFKyKOt8qI=",
+  "ID": "5f423562-aca1-53c3-e121-cb0eb2ea1cd3",
+  "ModifyIndex": 7,
+  "Name": "my-app-policy",
+  "Rules": "key \"\" { policy = \"read\" } key \"foo/\" { policy = \"write\" } key \"foo/private/\" { policy = \"deny\" } operator = \"read\""
+}
+```
+
+### Linking Policies to Tokens
+
+A policy that has been implemented must still be linked to a token before the policy has an effect. A service or agent presents the token when interacting with resources in the network. The ACL system processes evaluate the policies linked to the token to determine if the requester has access to the requested resource.
+
+The person responsible for administrating ACLs can use the command line or call the API endpoint to link policies to tokens. Tokens can also be generated dynamically from an external system using Consul's [auth methods](/docs/security/acl/auth-methods) functionality.
+
+Refer to the [tokens documentation](/docs/security/acl/acl-tokens), as well as the [ACL tutorial](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production#create-the-agent-token), for details about creating and linking policies to tokens.
+
+## Policy Attributes
```

Comment:
I still think document the attributes of a policy is a mistake. These are internal implementation details that could change. We already have the public "attributes" documented in two places, once in the API docs reference, and again in the CLI reference. This third copy make the docs more difficult to maintain, and doesn't really help understanding the concepts.  My suggestion would be to delete this section, and move any important information to one of the existing references.